Gera/Ejercicios UBA/Funciones/Ejercicio9.py

```diff
@@ -0,0 +1,72 @@
+"""Se leen 300 datos (usar constantes para poder achicar esta cantidad) que
+representan el peso de la misma cantidad de niños que hay internados en un
+hospital. Se desea confeccionar la siguiente tabla:
+- Entre 0,000 y 10,000 kg. hay ............... niños.
+- Entre 10,001 y 20,000 kg. hay ............. niños.
+- Entre 20,001 y 30,000 kg. hay ............. niños.
+- Más de 30,000 kg. hay .................... niños. """
+
+import random
+
+
+def ingreso_cantidad_ninios():
+    ninios = int(input("ingresar la cantidad de niños totales: "))
+    return ninios
+
+
+def generar_ingreso_peso_aleatorio(cantidad_ninios):
+
+    peso = []
+    for i in range(0, cantidad_ninios):
+        peso.append(random.uniform(float(0.000), float(40.000)))
+
+    return peso
+
+
+def contar_ninios_por_peso(cantidad_ninios, pesajes):
+    cantidad_a = 0
+    cantidad_b = 0
+    cantidad_c = 0
+    cantidad_d = 0
+    i = 0
+
+    while i <= cantidad_ninios - 1:
```

Comment:
Podés dejarlo con un while pero un for sería más adecuado. Vos tenés un rango predefinido para recorrer. De 0 a la cantidad de niños.

Reply:
Si. Puede ser de las dos maneras, pero me resulta más visible con un while.